.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Go MCP SDK Dev Container",
+    "image": "mcr.microsoft.com/devcontainers/go",
+    "features": {
+        "ghcr.io/devcontainers/features/git-lfs:1": {}
```

Comment:
Why is this needed?

Reply:
So [features](https://containers.dev/implementors/features/) are essentially units of code for adding tooling for the Dev Container.

When running the project for the first time, there was error that Git LFS was required, and thereby I had added it.

If it isn't an issue, this can be potentially removed as well.